src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.util.ts

```diff
@@ -23,6 +23,16 @@ export const getNodeDetailTabs = (nodeType: NodeType) => {
     }
 }
 
+export const getNodeDetailResourceBrowserTabs = (nodeType: NodeType) => {
```

Comment:
We can reuse the existing function by passing props instead of creating this one

Reply:
Made the required changes.